clarity_ext/extensions.py

```diff
@@ -272,20 +275,29 @@ def _run(self, config, path, pid, module, artifacts_to_stdout, disable_context_c
             context.logger.log("Finished running script '{}'".format(module))
             context.commit()
         except UsageError as e:
-            # Commit in order to upload step log
-            # Ordinary files and objects shouldn't have been moved to upload queue,
-            # since the scipt have been interrupted by the usage error.
-            context.commit()
```

Comment:
I think this is a good change. Although the comment says that it's supposed to not have anything in the upload queue. I assume that might have been true at some point but then changed after some refactoring. Better to be sure and just commit the step log!